modules/ztp-sno-du-reducing-resource-usage-with-olm-pprof.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-reducing-resource-usage-with-olm-pprof_{context}"]
+= Operator Lifecycle Manager
+
+{sno-caps} clusters that run DU workloads require consistent access to CPU resources. Operator Lifecycle Manager (OLM) periodically gathers performance data from Operators, which causes a spike in CPU usage. The following `ConfigMap` custom resource (CR) disables the gathering of Operator performance data by OLM.
```

Comment:
```suggestion
{sno-caps} clusters that run DU workloads require consistent access to CPU resources. The Operator Lifecycle Manager (OLM) collects performance data from operators at regular intervals, resulting in an increase in CPU utilisation. The following `ConfigMap` custom resource (CR) disables the collection of Operator performance data by the OLM.
```

Reply:
DU ??

Reply:
Great rewrite, thanks! FYI - OLM should have "the" before it according to OCP [glossary](https://github.com/openshift/openshift-docs/blob/30dc46d020d38222dea3277536910424be756775/contributing_to_docs/term_glossary.adoc#operator-lifecycle-manager-olm).